runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpSnapshotTotal.java

```diff
@@ -32,7 +32,7 @@
 	private final int xpGainedInSession;
 	private final int xpPerHour;
 
-	public static XpSnapshotTotal zero()
```

Comment:
What's with all these unrelated access modifiers?

Reply:
Discussed in Discord--given `XpTrackerService` is used in other plugins, I've reverted its modifier change. The others are only used internally and should be fine.